component/FollowList.tsx

```diff
@@ -0,0 +1,46 @@
+import ListLayout from "@/common/ListLayout";
+import { FollowInfoDataTypes } from "@/type/user";
+import * as styles from "../style/FollowList/FollowList.css";
+
+const FollowList = ({
+  followInfoData,
+}: {
+  followInfoData: FollowInfoDataTypes;
+}) => {
+  const { followingData, followersData } = followInfoData;
+
+  const matchedList = followersData.filter((follower) => {
+    return followingData.some(
+      (following) => following.login === follower.login
+    );
+  });
+
+  const unfollowingList = followersData.filter((follower) => {
+    return !matchedList.includes(follower);
+  });
+
+  const user = [matchedList, unfollowingList];
```

Comment:
user를 위와 같은 array 형식으로 사용할거라면,
matchedList와 unfollowingList를 return 해주는 함수를 정의해줘도 좋을것 같아요!
```suggestion
  const getUserList = useMemo( () => {
    const matchedList = followersData.filter((follower) => {
    return followingData.some(
      (following) => following.login === follower.login
    );
  });

  const unfollowingList = followersData.filter((follower) => {
    return !matchedList.includes(follower);
  });
  
  return [matchedList, unfollowingList]
  }, [followersData, followingData])
```